compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -702,98 +687,158 @@ struct QuantizeBias final : public luci::CircleNodeMutableVisitor<void>
   loco::DataType output_type;
   QuantizationGranularity granularity;
 
-  // Quantize bias node
-  void visit(luci::CircleNode *node)
+private:
+  // Return a quantized bias node
+  CircleConst *quantized_bias(CircleNode *input, const CircleNode *weight, CircleNode *bias)
```

Comment:
Ah sorry for confusion :( 
Then how about using template?

```suggestion
  template <uint32_t I, uint32_t W, uint32_t B>
  CircleConst *quantized_bias(CircleNode *node)
  {
    LOGGER(l);
    INFO(l) << "QuantizeBias visit node: " << node->name() << std::endl;
    
    auto input = loco::must_cast<CircleNode *>(node->arg(I));
    auto weight = loco::must_cast<const CircleNode *>(node->arg(W));
    auto bias = dynamic_cast<CircleConst *>(node->arg(B));
    
    // If bias cannot be quantized, return itself
    if (bias == nullptr || is_quantized(bias))
      return nullptr
```

If so, this can be applied as following
```cpp
  void visit(luci::CircleConv2D *node)
  {
    if (auto new_bias = quantized_bias<0, 1, 2>(node))
      node->bias(new_bias);
  }

  void visit(luci::CircleDepthwiseConv2D *node)
  {
    auto new_bias = quantized_bias<0, 1, 2>(node);
    node->bias(new_bias);
  }

  void visit(luci::CircleTransposeConv *node)
  {
    auto new_bias = quantized_bias<2, 1, 3>(node);
    node->bias(new_bias);
  }

  void visit(luci::CircleFullyConnected *node)
  {
    auto new_bias = quantized_bias<0, 1, 2>(node);
    node->bias(new_bias);
  }
```

Reply:
> If there are exceptional cases that bias is not arg(2), we can define them in it's visit function :)

Well, I think making a function w/ such an assumption is not good. It would be easy to misuse the function.

It would be better to make another function to collect input, weights, bias.